x-pack/plugins/osquery/public/editor/index.tsx

```diff
@@ -5,8 +5,10 @@
  * 2.0.
  */
 
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useCallback, useRef } from 'react';
 import useDebounce from 'react-use/lib/useDebounce';
+import styled from 'styled-components';
+import { EuiResizeObserver } from '@elastic/eui';
```

Comment:
maybe we could use https://elastic.github.io/eui/#/layout/resizable-container ?

Reply:
Interesting, I'll see if it changes anything, thanks! :)

Reply:
Thanks for suggesting this, I checked the ResizableContainer and what I found out is that it works if we want to resize 2 elements within one container. In our case we resize just one. So useResizeObserver seems to be the way.